apps/web/lib/api/links/process-link.ts

```diff
@@ -221,9 +260,11 @@ export async function processLink({
     };
   }
 
+  let expiresAt: Date | null = null;
+
   // expire date checks
-  if (expiresAt) {
-    const datetime = parseDateTime(expiresAt);
+  if (expiresAtRaw) {
+    const datetime = parseDateTime(expiresAtRaw);
```

Comment:
Do we still need this change?

Reply:
Just updated this to be less complicated ([commit](https://github.com/dubinc/dub/pull/631/commits/39b151a7470c64d9edbf0dfc9d0eca86ef0cba94))